README.md

```diff
@@ -1,2 +1,223 @@
 # fancy-todo
-Membuat website untuk mencatat hal - hal menarik untuk dilakukan
+Membuat website untuk mencatat hal - hal menarik untuk dilakukan. App ini memiliki fitur:
+* RESTful endpoint untuk operasi CRUD
+
+## RESTful endpoints
+### POST /todos
+
+> Menambahkan Todos
+
+_Request body_
+```json
+{
+  "title": "judul untuk hal yang ingin dilakukan",
+  "description": "deskripsi untuk hal yang ingin dilakukan",
+  "status": "<default: boolean(false)>",
+  "due_date": "tanggal untuk hal yang ingin dilakukan"
+}
+```
+
+_Response (201 - Created)_
+```json
+{
+  "id": "<default given by system>",
+  "title": "<posted title>",
+  "description": "<posted description>",
+  "status": "<posted status || default: boolean(false)>",
+  "due_date": "<posted date>",
+  "updatedAt": "<default given by system>",
+  "createdAt": "<default given by system>"
+}
+```
+
+_Response (400 - Bad Request)_
+```json
+{
+  "message": "<error message by system generated>"
+}
+```
+
+_Response (500 - Internal Server Error)_
+```json
+{
+  "message": "Internal Server Error"
+}
+```
+
+### GET /todos
+
+> Menampilkan semua Todos
+
+_Request body_
+```json
+{
+  "not needed"
+}
```

Comment:
tidak usah dibungkus object, langsung not needed aja

Reply:
siap kak saya revisi lagi documentationnya